<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileServer</class>
 <widget class="QWidget" name="FileServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FileServer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件服务器Demo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tableWidget">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="clientDownloadWidget">
        <attribute name="title">
         <string>客户端文件下载</string>
        </attribute>
        <widget class="QPushButton" name="selectFilebtn">
         <property name="geometry">
          <rect>
           <x>241</x>
           <y>30</y>
           <width>80</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="fileEdit">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>461</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>190</y>
           <width>701</width>
           <height>321</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>150</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="text">
          <string>信息框</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearMsg">
         <property name="geometry">
          <rect>
           <x>750</x>
           <y>470</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Serverbtn">
         <property name="geometry">
          <rect>
           <x>344</x>
           <y>30</y>
           <width>80</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>启动服务</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>13</x>
           <y>69</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件路径</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>14</x>
           <y>114</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="text">
          <string>发送进度</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="sendProgressBar">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>112</y>
           <width>571</width>
           <height>41</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="fileSizeLabel">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>70</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>大小：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="resetProgress">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>112</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="setting">
        <attribute name="title">
         <string>设置</string>
        </attribute>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>471</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="text">
             <string>监听端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="portEdit">
            <property name="font">
             <font>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="text">
             <string>8888</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
